selfdrive/car/toyota/values.py

```diff
@@ -39,28 +39,29 @@ class CAR:
   LEXUS_CTH = "LEXUS CT 200H 2018"
   RAV4H_TSS2 = "TOYOTA RAV4 HYBRID 2019"
   LEXUS_NXH = "LEXUS NX300H 2018"
+  LEXUS_NX = "LEXUS NX300 2018"
 
 
 # addr: (ecu, cars, bus, 1/freq*100, vl)
 STATIC_MSGS = [
-  (0x128, Ecu.dsu, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.LEXUS_NXH, CAR.RAV4, CAR.COROLLA, CAR.AVALON), 1,   3, b'\xf4\x01\x90\x83\x00\x37'),
+  (0x128, Ecu.dsu, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.LEXUS_NXH, CAR.LEXUS_NX, CAR.RAV4, CAR.COROLLA, CAR.AVALON), 1,   3, b'\xf4\x01\x90\x83\x00\x37'),
```

Comment:
remove all the additions to STATIC_MSGS. this was for old hardware and will be removed soon.